Resources/Prototypes/Adventure/Maps/Pools/Adventure.yml

```
@@ -6,7 +6,7 @@
   - AdventureAspid      #min 20 max 55
 # - AdventureAvrite     #min 40 max N/A
   - AdventureBagel      #min 30 max 120
-#   - AdventureBoxNew        #min 25 max 120 #rework (too big, original min 50) # курсед карта, надо чинить
+  - AdventureBoxNew        #min 25 max 120 #rework (too big, original min 50) # курсед карта, надо чинить
```

```suggestion
  - AdventureBoxNew        #min 25 max 120 #rework (too big, original min 50) 
```
Balls, карта же не курсед больше

---

:Jokerge